.gitignore

```diff
@@ -17,3 +17,6 @@ dist/
 
 # Test coverage reports
 coverage/
+
+# vscode artifacts
+.vscode/
```

Comment:
How does this relate to the changes in this PR? I never used VSCode... does it generate these files automatically for every project? Or what are they for?

Reply:
It doesn't directly relate to the changes. This dir is where VSCode keeps some per project files (such as build task definitions etc)

Reply:
I think this belongs into you [own global gitignore ](https://stackoverflow.com/questions/7335420/global-git-ignore) config, since it is quite specifc to your editor setup :). 

```suggestion
```

Reply:
Totally forgot about the global gitignore, thanks!